### ISSUE TITLE
More refactoring of GLAM ETL for Glean data

### ISSUE THREAD

**Issue by acmiyaguchi** [mozilla/bigquery-etl#897]
There are several breaking changes that I would like to make to the GLAM ETL
pipeline for Fenix.

* Change the naming convention for queries
* Change query names for consistency
* Rename dimensions to be closer to `client_info`

## Naming convention for queries

Currently the naming convention is `{app_name}_*`. I would like to change this
to `{app_id}__*`. The `app_id` is used in BigQuery for creating the dataset name
e.g. `org_mozilla_fenix_stable`, where the convention is `{app_id}_stable`. The
double underscore is so the table name can be unambiguously parsed.

## Changing query names for consistency

Here's the proposed diff, based on the current aggregates.

```diff
{app_id}__clients_daily_histogram_aggregates_{ping_type}_v1
{app_id}__clients_daily_scalar_aggregates_{ping_type}_v1
{app_id}__clients_histogram_aggregates_v1
-{app_id}__clients_histogram_bucket_counts_v1
+{app_id}__histogram_bucket_counts_v1
-{app_id}__clients_histogram_probe_counts_v1
+{app_id}__histogram_probe_counts_v1
{app_id}__clients_scalar_aggregates_v1
-{app_id}__clients_scalar_bucket_counts_v1
+{app_id}__scalar_bucket_counts_v1
-{app_id}__clients_scalar_probe_counts_v1
+{app_id}__scalar_probe_counts_v1
{app_id}__extract_probe_counts_v1
{app_id}__extract_user_counts_v1
{app_id}__histogram_percentiles_v1
{app_id}__latest_versions_v1
{app_id}__scalar_percentiles_v1
{app_id}__view_clients_daily_histogram_aggregates_v1
{app_id}__view_clients_daily_scalar_aggregates_v1
{app_id}__view_probe_counts_v1
{app_id}__view_user_counts_v1
```

Applied and sorted, it looks as follows:

```diff
{app_id}__clients_daily_histogram_aggregates_{ping_type}_v1
{app_id}__clients_daily_scalar_aggregates_{ping_type}_v1
{app_id}__clients_histogram_aggregates_v1
{app_id}__clients_scalar_aggregates_v1
{app_id}__extract_probe_counts_v1
{app_id}__extract_user_counts_v1
{app_id}__histogram_bucket_counts_v1
{app_id}__histogram_percentiles_v1
{app_id}__histogram_probe_counts_v1
{app_id}__latest_versions_v1
{app_id}__scalar_bucket_counts_v1
{app_id}__scalar_percentiles_v1
{app_id}__scalar_probe_counts_v1
{app_id}__view_clients_daily_histogram_aggregates_v1
{app_id}__view_clients_daily_scalar_aggregates_v1
{app_id}__view_probe_counts_v1
{app_id}__view_user_counts_v1
```

To recap:

* `bucket_counts` and `probe_counts` are not client level aggregates and are renamed in a similar convention to `percentiles`.

Edit: applied changes integrated into #870

## Changing attribute names to match Glean `client_info`

Currently the attributes are as follows:

```bash
client_id
ping_type
os
app_version
app_build_id
channel
```

Since these are just variables that can be changed fairly easy, I think these should match the ones in the [client_info](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section) section, with the exception of `app_display_version`. If we are changing other variables, it would be fine to rename this to be more descriptive. 

```bash
client_id
ping_type
os
app_major_version
app_build
app_channel
```

**Comment by emtwo** [comment]
These all seem like reasonable and helpful changes to me.

**Comment by emtwo** [comment]
Is this complete now @acmiyaguchi ?

**Comment by acmiyaguchi** [comment]
Yes, this should be complete with #999, #1002, and #1011 .
